doc/widgets/chaining.rst

```diff
@@ -0,0 +1,57 @@
+Chaining
+========
+
+.. figure:: icons/chaining.png
+   :alt: Chaining widget icon
+
+   Chaining widget icon
+Profiles objects of one type in the latent space of another object type
+through chaining of latent matrices along paths in a data fusion graph.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Fitted Fusion Graph**
+
+Fitted collective latent data model.
```

Comment:
This dedentation breaks the list. See how it renders: https://github.com/biolab/orange3-datafusion/pull/19/files?short_path=cfbfdec#diff-cfbfdec10a0c528d24b648cd66e2b2f2